docs/clients/uavcan_node.rst

```diff
@@ -0,0 +1,132 @@
+| 8      | error_warning_flag     | get            | uint8     | Bool             | This flag indicates if the error warning limit has been reached. 1 means there has been enough errors to trigger a warning.                                                                                                                                                                                             |
++--------+------------------------+----------------+-----------+------------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| 9      | telemetry_frequency    | get, set, save | uint32    | Hz               | The frequency with which telemetry will be sent to the DroneCAN bus                                                                                                                                                                                                                                                     |
++--------+------------------------+----------------+-----------+------------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| 10     | bit_rate               | get, set, save | uint32    | Bit/s            | The bitrate used by the DroneCAN bus.                                                                                                                                                                                                                                                                                   |
++--------+------------------------+----------------+-----------+------------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| 11     | bypass_arming          | get, set, save | uint8     | Bool             | This setting allows the module to bypass arming with DroneCAN throttle messages. DroneCAN messages will not be impacted by the arming state, will not cause arming state transistions, and will use the zero behavior setting. This is useful for using the 'old-style' DroneCAN support.                               |
```

Comment:
'old-style' doesn't really mean anything to anyone outside of us. Maybe we should put an actual firmware number where this changed? @iq-fred ?

Reply:
I mean you can still use this 'old-style' on the latest firmware if you have bypass arming, so there isn't really a version where it totally went away. And the 8108 still has this set to true by default. So a version is tricky. if the 'old-style' is problematic, we should probably just lose this sentence altogether